app/assets/javascripts/retros.js

```diff
@@ -45,19 +42,23 @@ document.addEventListener("DOMContentLoaded", function(event) {
       timeLeft = timeLeft - 1000;
       if (timeLeft >= 0) {
         let newValue = getTimerValue();
-        document.getElementById("time").innerText = newValue;
+        timeText.text(newValue);
       }
       if (timeLeft === 0) {
         clearInterval(timeLeft);
       }
     }, 1000);
   }
 
-  startNewRetroButton.addEventListener("click", function() {
-    startNewRetroButton.style.display = "none";
+  startNewRetroButton.on("click", function() {
```

Comment:
Since ES6, you can make use of [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), that has a shorter syntax and are more suitable for declaring non-method functions. 
Maybe you like them 😄